### PR TITLE
feat(two-searched-pokemon-screen): adds labels to user and enemy pokemon

### LINKED ISSUE #60: Signs showing which Pokémon is the searched and the enemy
**Is your feature request related to a problem? Please describe.**
Both Pokémon show up on the vs screen without an indicator of which is which

**Describe the solution you'd like**
Have small indicators saying which Pokémon is which

### PR DESCRIPTION
**Pull request by Wolftownn**
## Changes
_List Changes Introduced by this PR_
1. Adds labels to user's Pokemon, and enemy Pokemon in the versus search screen.

## Purpose
These labels give the user an indication of which Pokemon he is searching for in the versus screen.

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #60

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Looks really good in App

File: pokemon-app/src/components/SinglePokemon.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ const SinglePokemon = ({
                       ))}
                 </ol>
               </div>
+              <div className={hasEnemy ? "user-poke" : "user-poke-none"} >Your Pokemon</div>
```
Comment (krferrell) [inline pokemon-app/src/components/SinglePokemon.js:77]: Solid use of ternary operator to change the class.

**Review by JayDampitan — APPROVED** [review]
Great styling Jorge, keep up the good work!

File: pokemon-app/src/components/SinglePokemon.js (hunk 1 above)
Comment (BlackCubes) [inline pokemon-app/src/components/SinglePokemon.js:77]: Nice ternary conditional to generate the classNames

**Review by krferrell — APPROVED** [review]
Looking great here!

**Review by BlackCubes — APPROVED** [review]
Everything looks good and clean!

**Pull request merged**